### PR TITLE
Cleanup docstrings

### PR DESCRIPTION
**Pull request by timhoffm**
## PR Summary

Docstring cleanups:

- (Half)-sentences should always end with a period (PEP-257: The docstring is a phrase ending in a period; and numpydoc style)
- pydocstyle D200: One-line docstring should fit on one line with quotes.
- No trailing blank lines in docstrings.
- Blank line after class docstring. (PEP-257: Insert a blank line after all docstrings (one-line or multi-line) that document a class -- generally speaking, the class's methods are separated from each other by a single blank line, and the docstring needs to be offset from the first method by a blank line.)
- Some other misc cleanups.

### CODE REVIEW

File: lib/matplotlib/backend_tools.py [hunk 1]
```diff
@@ -743,7 +743,7 @@ class ToolHome(ViewsPositionsBase):
 
 
 class ToolBack(ViewsPositionsBase):
-    """Move back up the view lim stack"""
+    """Move back up the view limiits stack."""
```
Comment (dstansby) [inline lib/matplotlib/backend_tools.py]:
```suggestion
    """Move back up the view limits stack."""
```

File: lib/matplotlib/backend_tools.py [hunk 2]
```diff
@@ -1132,11 +1132,10 @@ def add_tools_to_container(container, tools=default_toolbar_tools):
...
     ----------
     container : Container
-        `backend_bases.ToolContainerBase` object that will get the tools added
+        `backend_bases.ToolContainerBase` object that will get the tools added.
     tools : list, optional
-        List in the form
-        [[group1, [tool1, tool2 ...]], [group2, [...]]]
-        Where the tools given by tool1, and tool2 will display in group1.
+        List in the form ``[[group1, [tool1, tool2 ...]], [group2, [...]]]``
+        where the tools ``[tool1, and tool2, ...]`` will display in group1.
```
Comment (dstansby) [inline lib/matplotlib/backend_tools.py]:
```suggestion
        where the tools ``[tool1, tool2, ...]`` will display in group1.
```

File: lib/matplotlib/patches.py [hunk 3]
```diff
@@ -3570,9 +3531,7 @@ def transmute(self, path, mutation_size, linewidth):
 
     @_register_style(_style_list, name="]-[")
     class BracketAB(_Bracket):
-        """
-        An arrow with a bracket(]) at both ends.
-        """
+        """An arrow with a bracket(]) at both ends."""
```
Comment (anntzer) [inline lib/matplotlib/patches.py]: space before parenthesis

File: lib/matplotlib/patches.py [hunk 4]
```diff
@@ -3604,9 +3563,7 @@ def __init__(self,
 
     @_register_style(_style_list, name="]-")
     class BracketA(_Bracket):
-        """
-        An arrow with a bracket(])  at its end.
-        """
+        """An arrow with a bracket(])  at its end."""
```
Comment (anntzer) [inline lib/matplotlib/patches.py]: idem

File: lib/matplotlib/patches.py [hunk 5]
```diff
@@ -3626,9 +3583,7 @@ def __init__(self, widthA=1., lengthA=0.2, angleA=None):
 
     @_register_style(_style_list, name="-[")
     class BracketB(_Bracket):
-        """
-        An arrow with a bracket([)  at its end.
-        """
+        """An arrow with a bracket([)  at its end."""
```
Comment (anntzer) [inline lib/matplotlib/patches.py]: idem

File: lib/matplotlib/patches.py [hunk 6]
```diff
@@ -3648,9 +3603,7 @@ def __init__(self, widthB=1., lengthB=0.2, angleB=None):
 
     @_register_style(_style_list, name="|-|")
     class BarAB(_Bracket):
-        """
-        An arrow with a bar(|) at both ends.
-        """
+        """An arrow with a bar(|) at both ends."""
```
Comment (anntzer) [inline lib/matplotlib/patches.py]: idem

**Comment by NelleV** [comment]
Thanks @timhoffm !

**Pull request merged**